Python/ibmcloudsql/SQLQuery.py

```diff
@@ -31,99 +30,175 @@
 import pyarrow
 import os
 import tempfile
+try:
+    from exceptions import RateLimitedException
+except Exception:
+    from .exceptions import RateLimitedException
+try:
+    from .cos import COSClient
+except Exception:
+    from cos import COSClient
+try:
+    from sql_magic import SQLMagic, format_sql, print_sql
+except Exception:
+    from .sql_magic import SQLMagic, format_sql, print_sql
+try:
+    from .catalog_table import HiveMetastore
+except Exception:
+    from catalog_table import HiveMetastore
+
+import logging
+logger = logging.getLogger(__name__)
+from functools import wraps
+import json
+from json import JSONDecodeError
+import getpass
+import requests
+import dateutil.parser
+from pprint import pformat
+from collections import namedtuple
+import inspect
+import sqlparse
+
+from enum import Enum
+
+def check_saved_jobs_decorator(f):
```

Comment:
Seems this decorator is only used once. Do you need it as decorator then? Can't it be folded simply into the single decorated method below?

Reply:
While it is currently being used at only one method, I suggest to keep it separately for the following reasons: the API is cleaner, the decorator can be applied to related APIs as needed, e.g. run(), submit() ...; the decorator can be expanded to support local file when SQL Query client is being used from a local machine or a machine that has no connection to Watson Studio project lib.